test/spec/refinements.js

```diff
@@ -111,7 +112,7 @@ test('hasRefinements(facet) should return true if the facet is refined.', functi
 
   // in complete honesty we should be able to detect numeric facets but we can't
   // t.throws(helper.hasRefinements.bind(helper, 'notAFacet'), 'not a facet');
-  expect(_.bind(helper.hasRefinements, null)).toThrow();
+  expect(helper.hasRefinements(null)).toBe(false);
```

Comment:
this was throwing for the wrong reason (bind was missing `, helper,`) and we stopped making this throw in v3